system/src/ble_control_request_channel.cpp

```diff
@@ -92,18 +95,6 @@ struct __attribute__((packed)) HandshakeHeader {
 // Device setup protocol version
 const unsigned PROTOCOL_VERSION = 0x02;
 
-// UUID of the control request service
-const uint8_t CTRL_SERVICE_UUID[] = { 0xfc, 0x36, 0x6f, 0x54, 0x30, 0x80, 0xf4, 0x94, 0xa8, 0x48, 0x4e, 0x5c, 0x01, 0x00, 0xa9, 0x6f };
```

Comment:
Let's keep this as-is. This looks clearer with the comments and named constants. `hal_ble_uuid` can be initialized from this data, or even higher level `BleUuid` can be used to simplify things.